components/Common/NavBar.tsx

```diff
@@ -23,6 +24,13 @@ const NavBar: FC = () => {
         <div className="flex item-center flex-shrink-0 cursor-pointer">
           <Image src={Logo} alt="MCSS logo" height={80} onClick={() => router.push('/')} />
         </div>
+        <div className={`${
+          (router.pathname === '/Events') ? '' : 'invisible'
+        }  m-5 w-full h-12 block flex-grow lg:flex lg:items-center lg:w-auto `}
+        >
+          <div className="bg-gray-200 h-full text-base text-gray-400 pl-3 py-3"><SearchIcon className="  " /></div>
```

Comment:
Is `className` field a requirement for `SearchIcon` component?

Reply:
Yeah, I think it was complaining that class Name is missing

Reply:
Alright sounds right